lib/models/team_member.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter/material.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'team_member.g.dart';
+
+@JsonSerializable()
+class TeamMember extends Equatable {
+  final String name;
```

Comment:
We might later need some "uuid" here.. When we later want to support "remove team member" since we could have more members with same name...

probably also later need some way to reference the profile image here